src/Nethermind/Nethermind.Specs/MainNetSpecProvider.cs

```diff
@@ -57,11 +58,13 @@ public void UpdateMergeTransitionInfo(long? blockNumber, UInt256? terminalTotalD
         public const long LondonBlockNumber = 12_965_000;
         public const long ArrowGlacierBlockNumber = 13_773_000;
         public const long GrayGlacierBlockNumber = 15_050_000;
-        public const ulong ShanghaiBlockTimestamp = ulong.MaxValue - 4;
+        public const ulong ShanghaiBlockTimestamp = ulong.MaxValue - 5;
+        public const ulong ShardingForkBlockTimestamp = ulong.MaxValue - 4;
```

Comment:
QQ: What is the reason with this `-1` or `-5`?

Reply:
It is may be to show the order of the forks for developers. I did the same just for consistency.